### DISCUSSION TITLE
App crash on startup

### DISCUSSION THREAD

**Discussion by manoj-paramesh-learngram-ai** [getsentry/sentry-react-native#2999] [Q&A]
Hi,

Using the react native 0.63.3, and using the sentry react native sdk v4 (4.15.0). Since we have auto linking, i have not linked it manually. Therefore just running the following commands 

```
yarn add @sentry/react-native@4
npx @sentry/wizard@1 -i reactNative -p ios android
npx pod-install
```
While the app runs fine in the ios simulator, the app crashes on startup in production build. Not sure what is wrong here exactly. Any help is greatly appreciated . The app works fine on android (Production build).

Crash log : https://pastebin.com/PTwjFRTr

**Comment by krystofwoldrich** [comment]
Hi,
thank you for reaching out, sadly I don't see anything useful in the crash log. 

Would you be able to share a minimal reproducible example?
